### PR TITLE
add spiderum presence v1.0.0

### PR DESCRIPTION
**Pull request by DawkinNguyen**
image

### CODE REVIEW

**Comment by DawkinNguyen** [comment]
> Fix the description

Fixed

**Comment by doomlerd** [comment]
vi_VN

File: websites/S/Spiderum/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Dawkin Nguyen",
+    "id": "414274321779916800"
+  },
+  "service": "Spiderum",
+  "description": {
+    "en": "Vietnamese writing community",
+    "vi": "Nền tảng chia sẻ - thảo luận dành cho người Việt. Nơi tập trung những nội dung thú vị và hữu ích nhất do người dùng đóng góp và bình chọn."
```
Comment (Bas950) [inline websites/S/Spiderum/dist/metadata.json:9]: [vi_VN](https://api.premid.app/v2/langFile/list)

Comment (DawkinNguyen) [reply websites/S/Spiderum/dist/metadata.json:9]: alright

**Pull request merged**